### PR TITLE
www-apps/radicale: bump to 2.1.12

### PR DESCRIPTION
**Pull request by m1027**
Signed-off-by: m1027 <m1027@posteo.net>

### CODE REVIEW

**Comment by ajakk** [comment]
[GLEP 76 says:](https://www.gentoo.org/glep/glep-0076.html#id26)

The sign-off must contain the committer's legal name as a natural person

This PR can't be merged without a real name in the signoff. As for the ID clash just pick an available one counting down from 500 from [here](https://github.com/gentoo/api-gentoo-org/blob/master/files/uid-gid.txt) and if necessary it can be at merge time.

File: www-apps/radicale/radicale-2.1.12.ebuild
Comment (ConiKost) [inline www-apps/radicale/radicale-2.1.12.ebuild:6]: Please test, if python3.8 and python3.9 is supported.

File: acct-user/radicale/radicale-0.ebuild
Comment (ConiKost) [inline acct-user/radicale/radicale-0.ebuild:9]: That won't work, as UID and GID 350 is already taken by user/group 'colord'.
Please submit a PR for reserving a free UID/GID at https://github.com/gentoo/api-gentoo-org.

File: acct-group/radicale/radicale-0.ebuild
Comment (ConiKost) [inline acct-group/radicale/radicale-0.ebuild:9]: That won't work, as UID and GID 350 is already taken by user/group 'colord'.
Please submit a PR for reserving a free UID/GID at https://github.com/gentoo/api-gentoo-org.

File: www-apps/radicale/radicale-2.1.12.ebuild
Comment (ConiKost) [inline www-apps/radicale/radicale-2.1.12.ebuild:4]: Please update eBuild to EAPI=7 while bumping it.

File: www-apps/radicale/radicale-2.1.12.ebuild
Comment (ConiKost) [inline www-apps/radicale/radicale-2.1.12.ebuild:56]: Remove such comments from ebuild please.

**Review by ConiKost — CHANGES_REQUESTED** [review]
Please also rebase your commit and make CI happy :-)

File: www-apps/radicale/radicale-2.1.12.ebuild
Comment (m1027) [reply www-apps/radicale/radicale-2.1.12.ebuild]: Tested against python3.8, works. No test and no time to test against python3.9 yet.
So I'd keep python3.7 + python3.8 for now.

File: acct-user/radicale/radicale-0.ebuild
Comment (m1027) [reply acct-user/radicale/radicale-0.ebuild]: Fixed by a separate PR for gentoo-api. New numbers are 328.

File: acct-group/radicale/radicale-0.ebuild
Comment (m1027) [reply acct-group/radicale/radicale-0.ebuild]: Fixed by a separate PR for gentoo-api. New numbers are 328.

File: www-apps/radicale/radicale-2.1.12.ebuild
Comment (m1027) [reply www-apps/radicale/radicale-2.1.12.ebuild]: Done.

File: www-apps/radicale/radicale-2.1.12.ebuild
Comment (m1027) [reply www-apps/radicale/radicale-2.1.12.ebuild]: Which commits are okay, which are not acceptable? Removed anyway.

File: www-apps/radicale/radicale-2.1.12.ebuild
Comment (m1027) [reply www-apps/radicale/radicale-2.1.12.ebuild]: One more on python3.9 compatibility:

One important way to run the radicale server is plugging it to apache via
mod_wsgi. However, mod_wsgi has no python3.9 support yet, only python3.8.

mod_wsgi has to be recompiled against the same python single target as
radicale. mod_wsgi with python3.7 complains when run against radicale using
python3.8.

So yeah, I left radicale on python3.7 and python3.8.

Comment (ajakk) [reply www-apps/radicale/radicale-2.1.12.ebuild]: If a user wants to use mod_wsgi with radicale they can ensure their PYTHON_SINGLE_TARGETs are the same. mod_wsgi isn't a dependency of radicale, so radicale's PYTHON_COMPAT can be bumped without actually hurting mod_wsgi.

Comment (m1027) [reply www-apps/radicale/radicale-2.1.12.ebuild]: > If a user wants to use mod_wsgi with radicale they can ensure their
> PYTHON_SINGLE_TARGETs are the same. mod_wsgi isn't a dependency of radicale,
> so radicale's PYTHON_COMPAT can be bumped without actually hurting mod_wsgi.

Last time I checked radicale as a stand alone server (without apache/mod_wsgi)
I stubled accross some significant security issues, mentioned by the
developer himself.

So, I am close to saying mod_wsgi *is* kind of a dependency (while of couse
not beeing an ideal solution, either).

At least, I have no testbed without mod_wsgi and personally would prefer to
wait for mod_wsgi to become compatible with python3.9 so I can test radicale
against python3.9, too.

Let me know in case there are reasons to hurry up, though.

Comment (juippis) [reply www-apps/radicale/radicale-2.1.12.ebuild]: If mod_wsgi is a dependency, why it's not listed in the ebuild? You can also add it as `optfeature`.

**Review by juippis — APPROVED** [review]

**Pull request closed without merging**
